pages/shoes/buy/[id].js

```diff
@@ -83,6 +84,7 @@ const StyledLinkContainer = styled.section`
   }
   a {
     text-decoration: none;
+    color: black;
   }
 `;
 
```

Comment:
I think on the buy-page I would add a disclaimer that the user is redirected to an external page. I was not expecting it, when I clicked on the button because I didn't really think about it. Maybe that would be a nice add-on